Tests/SVT-HEVC_FunctionalTests.py

```diff
@@ -102,7 +101,7 @@ def get_test_mode(mode):
 
 ##-------------- TEST MODE SPECIFIC SETTINGS -------------##
 if VALIDATION_TEST_MODE == 0:
-    ENC_MODES                       = [1,4,8,12]
+    ENC_MODES                       = [7]
```

Comment:
why removing all 4 options ?

Reply:
I would like the fast mode even faster. But I can review to [1,4,8,11]